documentation/en/source/pages/machine-images/builders/overview.rst

```diff
@@ -5,7 +5,7 @@
 Builders
```

Comment:
During the meeting with Jeff, we've talked about a new section "Builders Specification", to be consistent with "Migrations Specification" and "Templates specifications".

Reply:
ok